spec/controllers/tags_controller_spec.rb

```diff
@@ -40,6 +40,12 @@
       expect(tags.first.name).to eq 'tag'
       expect(tags.first.description).to eq 'tag description'
     end
+
+    it 'associates the new tag with an assessment' do
+      post :create, params: { tag: { name: 'tag', description: 'tag description' },
+                              assignment_id: assignment.id }
+      expect(Tag.find_by(name: 'tag', description: 'tag description').assessment).to_not be_nil
```

Comment:
Rather than just testing that the assessment isn't `nil`, you can easily change this test so that the assessment is actually the correct one.